sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/rntbd/RntbdClientChannelPool.java

```diff
@@ -623,6 +623,12 @@ private double computeLoadFactor() {
         for (Channel channel : this.availableChannels) {
 
             final RntbdRequestManager manager = channel.pipeline().get(RntbdRequestManager.class);
+
+            if (manager == null) {
```

Comment:
Is this the null manager change we talked about on the email with context to linkedin?

Reply:
yes